src/control/server/storage/scm/provider.go

```diff
@@ -117,6 +117,8 @@ type (
 	FormatResponse struct {
 		Mountpoint string
 		Formatted  bool
+		Mounted    bool
+		Mountable  bool
```

Comment:
Might be worthwhile to comment that mountable implies !mounted. Wasn't totally obvious to me that they were mutually exclusive until I read the code.